src/classes/tests/ContactTriggerTest.cls

```diff
@@ -1,21 +1,19 @@
 @IsTest
-public class ContactTriggerTest {
+private class ContactTriggerTest {
 
     @TestSetup
     static void makeData(){
```

Comment:
MINOR
Again, always add access modificator to the methods: `private`
Also, space missing after signature and before body.